Form/Type/ResetPasswordType.php

```diff
@@ -26,6 +27,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'options' => array('attr' => array('class' => 'password-field')),
             'first_options' => array('label' => 'Password'),
             'second_options' => array('label' => 'Repeat Password'),
+            'constraints' => new NotBlank(array(
```

Comment:
@ssitu-dayspring Should this constraint go on the User model object with the other constraints?